style.css

```diff
@@ -902,3 +928,177 @@ body {
     top: 65%;
   }
 }
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] You've really done a great work so far, the app is responsive but there is a little challenge you need to address. At `min-width: 768px` I expect to see all the elements in your app displayed properly. Please 🙏 ensure that all the sections are aligned properly. Thanks.

## For example, this is your header section @ `min-width: 768px`
![Screenshot from 2022-11-25 09-04-14 (1)](https://user-images.githubusercontent.com/103386183/206678599-56ea4915-ddb0-4f51-a42f-a4740f0dc029.png)